code/modules/ascent/ascent_rigs.dm

```diff
@@ -213,6 +213,21 @@
 		list("tramadol",            "tramadol",            /datum/reagent/tramadol,      80),
 		list("crystal brothime",    "crystal brothime",    /datum/reagent/crystaltram,   80)
 	)
+//Serpentid version of chemical injector
+/obj/item/rig_module/chem_dispenser/mantid/serpentid
+	name = "serpentid chemical injector"
+	desc = "A compact chemical dispenser of mantid design."
+	interface_name = "serpentid chemical injector"
+	interface_desc = "A compact chemical dispenser of mantid design."
+	icon = 'icons/obj/ascent.dmi'
+	icon_state = "injector"
+	charges = list(
+		list("inaprovaline", "inaprovaline", /datum/reagent/inaprovaline, 20),
+		list("dexalin plus", "dexalin plus", /datum/reagent/dexalinp,       20),
+		list("spaceacillin", "spaceacillin", /datum/reagent/spaceacillin,  20),
+		list("tramadol",   "tramadol",    /datum/reagent/tramadol,      20),
+		list("glucose",    "glucose",   /datum/reagent/nutriment/glucose,   20)
+	)
```

Comment:
This is fine, could probably shorten it down as it's a subtype of the above but as said, fine.